lib/components/T3Form/T3FormFieldList/T3FormFieldList.vue

```diff
@@ -5,16 +5,19 @@
       :key="field.identifier"
       :class="[`t3-form-field-${field.identifier}`, getCssClass(field, classes)]"
     >
-      <template v-if="field.type === 'Fieldset'">
-        <T3FormFieldset :field="field">
+      <template v-if="field.fieldlist">
+        <component
+          :is="getComponentField(field)"
+          :field="field"
+        >
```

Comment:
What do you think about removing the v-if statement from a template in line 8 and the second block with the dynamic component? You can put this condition in the recursive `T3FormFieldList` slot.